forms/common-themes/src/main/resources/theme/base/admin/resources/partials/client-detail.html

```diff
@@ -207,7 +207,7 @@
 
                 <div class="col-sm-6">
                     <div class="input-group" ng-repeat="(i, redirectUri) in client.redirectUris track by $index">
-                        <input class="form-control" ng-model="client.redirectUris[i]">
+                        <input class="form-control" ng-model="client.redirectUris[i]" id="{{'redirectUri_' + i}}">
```

Comment:
Adding IDs like this is not great. It's also brittle from a testing perspective as you rely on the ordering. In this case XPath and selecting on value is better.